package.json

```diff
@@ -6,7 +6,6 @@
   "type": "module",
   "scripts": {
     "compile": "tsc",
-    "postinstall": "npm run compile",
```

Comment:
README doesn't mention this because it's assumed that install does it, so we should probably add it as an additional step there